yarn-project/txe/src/txe_service/txe_service.ts

```diff
@@ -210,26 +212,56 @@ export class TXEService {
   // PXE oracles
 
   getRandomField() {
```

Comment:
Once this is more stable it would be good to create a proxy or something to avoid all of this boilerplate!